ydb/core/protos/config.proto

```diff
@@ -1549,21 +1549,30 @@ message TTracingConfig {
         optional TSelectors Scope = 1;
         optional float Fraction = 2;
         optional uint32 Level = 3;
-        optional uint32 MaxRatePerMinute = 4;
-        optional uint32 MaxBurst = 5;
+        optional uint64 MaxRatePerMinute = 4;
+        optional uint64 MaxBurst = 5;
     }
 
     message TExternalThrottlingScope {
         optional TSelectors Scope = 1;
-        optional uint32 MaxRatePerMinute = 2;
-        optional uint32 MaxBurst = 3;
+        optional uint64 MaxRatePerMinute = 2;
+        optional uint64 MaxBurst = 3;
+    }
+
+    message TUploaderConfig {
+        optional uint64 MaxSpansPerSecond = 1;
+        optional uint64 MaxSpansInBatch = 2;
+        optional uint64 MaxBytesInBatch = 3;
+        optional uint64 MaxBatchAccumulationMilliseconds = 4;
+        optional uint32 SpanTtlSeconds = 5;
```

Comment:
What is "span TTL"? I'd expect time the span is alive in Jaeger, after which it is deleted. Here it actually means approximate time after span is deleted unless sent to collector.